modules/ossm-config-otel-kiali.adoc

```diff
@@ -23,22 +23,18 @@ After {KialiProduct} is integrated with {DTProductName}, you can view distribute
 ----
 spec:
   external_services:
-    grafana:
-      enabled: true # <1>
-      in_cluster_url: http://grafana-istio-system.apps-crc.testing/ # <2>
-      url: http://grafana-istio-system.apps-crc.testing/
     tracing:
-      enabled: true
+      enabled: true #<1>
       provider: tempo
       use_grpc: false
       in_cluster_url: http://tempo-sample-query-frontend.tempo:3200
-      url: https://tempo-sample-query-frontend-tempo.apps-crc.testing
-      tempo_config: # <3>
+      url: https://tempo-sample-query-frontend-tempo.apps-crc.testing #<2>
+      tempo_config:
```

Comment:
All the tempo config won't be needed, it is used to create the Tempo datasource for the Grafana url